Header header
uint16[100] team_id
uint16 team_num
